projects/optic/src/commands/ci/comment/common.ts

```diff
@@ -80,9 +101,11 @@ ${
 <thead>
 <tr>
 <th>API</th>
-<th>Operation Changes</th>
-<th>Checks</th>
+<th>Changes</th>
+<th>Rules</th>
 ${anyCompletedHasWarning ? '<th>Warnings</th>' : ''}
+<th>Tests</th>
```

Comment:
Should we maybe only render this if there's at least 1 capture run? Or maybe add instructions on how to get started with capture here

If we do the second, we'll need to add some information about how to convert from the `diff-all` or `GHA` or `gitlab` pipeline to `run`